tests/scripts/test_scripts.py

```diff
@@ -436,35 +421,6 @@ def test_train_adversarial(tmpdir, named_configs, command):
     _check_train_ex_result(run.result)
 
 
-@pytest.mark.parametrize("command", ("airl", "gail"))
-def test_train_adversarial_warmstart(tmpdir, command):
```

Comment:
Why are we removing this test? Do we no longer support warmstarting?

Reply:
Good catch! I added it back!